app/src/main/res/layout/fragment_details.xml

```diff
@@ -0,0 +1,158 @@
+        tools:text="@tools:sample/lorem" />
+
+    <ImageButton
+        android:id="@+id/favouritesButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dimen_26_dp"
+        android:layout_marginEnd="@dimen/dimen_16_dp"
+        android:background="@null"
+        android:src="@drawable/ic_favourites_unchecked"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/movieTitleTextView"
+        app:layout_constraintTop_toBottomOf="@id/posterImageView" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/ratingTextView"
+        style="@style/smallMontserrat_Regular"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dimen_16_dp"
+        android:layout_marginTop="@dimen/dimen_16_dp"
+        android:layout_marginEnd="@dimen/dimen_8_dp"
+        android:background="@drawable/textview_background"
+        android:drawableStart="@drawable/ic_star"
+        android:drawablePadding="@dimen/dimen_4_dp"
+        android:paddingHorizontal="@dimen/dimen_10_dp"
+        android:paddingVertical="@dimen/dimen_4_dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/movieTitleTextView" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/categoryTextView"
+        style="@style/smallMontserrat_Regular"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dimen_8_dp"
+        android:layout_marginTop="@dimen/dimen_16_dp"
+        android:layout_marginEnd="@dimen/dimen_8_dp"
+        android:background="@drawable/textview_background"
+        android:paddingHorizontal="@dimen/dimen_10_dp"
+        android:paddingVertical="@dimen/dimen_4_dp"
+        app:layout_constraintStart_toEndOf="@id/ratingTextView"
+        app:layout_constraintTop_toBottomOf="@id/movieTitleTextView"
+        tools:text="@tools:sample/lorem" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/durationTextView"
+        style="@style/smallMontserrat_Regular"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dimen_8_dp"
+        android:layout_marginTop="@dimen/dimen_16_dp"
+        android:layout_marginEnd="@dimen/dimen_8_dp"
+        android:background="@drawable/textview_background"
+        android:drawableStart="@drawable/ic_clock"
+        android:drawablePadding="@dimen/dimen_4_dp"
+        android:paddingHorizontal="@dimen/dimen_10_dp"
+        android:paddingVertical="@dimen/dimen_4_dp"
+        app:layout_constraintStart_toEndOf="@id/categoryTextView"
+        app:layout_constraintTop_toBottomOf="@id/movieTitleTextView"
+        tools:text="@tools:sample/lorem" />
```

Comment:
You can use other tools text to match original content, such as `"@tools:sample/date/hhmm"` or something like that.

Reply:
done